content/04.results.md

```diff
@@ -41,3 +41,19 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Discriminative Model Calibration
+
+![
+Calibration plots for the discriminative model.
+A perfectly calibrated model would follow the dashed diagonal line.
+The blue line represents the predictions after calibration and the green line shows predictions after calibration. 
+](https://raw.githubusercontent.com/danich1/snorkeling/86037d185a299a1f6dd4dd68605073849c72af6f/figures/model_calibration_experiment/model_calibration.png){#fig:discriminative_model_calibration}
+
+Even high performing deep learning models are subject to poor calibration. Typically, these models are overconfident in their predictions [@arxiv:1706.04599; @arxiv:1807.00263].
```

Comment:
```suggestion
Even deep learning models with good AUROC and AUPR statistics can be subject to poor calibration.
Typically, these models are overconfident in their predictions [@arxiv:1706.04599; @arxiv:1807.00263].
```
I think this may define what you mean more clearly than "high performing"